static/app/components/stories/sizingWindow.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from '@emotion/styled';
```

Comment:
Since this is used in replays and in stories, shouldn't this be in the base `/components` directory?